index.js

```diff
@@ -13,7 +14,43 @@ function checkLicense (path) {
     });
 }
 
-module.exports = function (options) {
+function ellipsis (text, limit) {
+    if (!text || text.length <= limit) {
+        return text;
+    }
+    return text.slice(0, limit - 3) + '...';
+}
+
+function renderAsJsDocs (data) {
+    return data.map(({ pkg, lic }) => [
+        '/*!',
+        ` * @package ${pkg.name}`,
+        ` * @version ${pkg.version}`,
+        ` * @license ${lic.licenses}`,
+        ` * @author ${lic.publisher}`,
+        ` * @url ${lic.repository || lic.email}`,
+        ' */'
+    ].join('\n')).join('');
+}
+
+function renderAsTable (data) {
+    let rows = data.map(({ pkg, lic }) => {
+        let source = ellipsis(lic.repository || lic.email, 65);
+        return [pkg.name, pkg.version, lic.licenses, ellipsis(lic.publisher, 30), source];
+    });
+    rows.sort(([a], [b]) => a.localeCompare(b));
+    rows.unshift(['Name', 'Version', 'License(s)', 'Publisher', 'Source']);
+
+    return [
+        '/*!',
+        ' * Bundled Libraries',
```

Comment:
Not sure I like the phrase "Bundled Libraries". Perhaps something like "NPM Package Licenses"?

Reply:
I think that “Bundled Libraries” communicate better that the names libraries are _bundled_ (included) in the source. “NPM Package Licenses” is unclear to me; licenses of what packages? Those bundled in the file, used to build this file, or external packages that this file depends on?

Reply:
Hmm good point. "Bundled NPM Dependencies/Packages/Modules"?

Just the word "Libraries" I'm mostly reluctant to use, as it's not consistent with NPM terminology. :)

Reply:
Hmm, maybe make it configurable? :smile: But we have to decide about the default anyway. It’s hard to decide but I think that I prefer “Bundled npm Packages” (“npm” is actually not an acronym, it should be written as “npm”, not “NPM”).

Reply:
No need for configuration I don't think. Just having a look over the npm website, and I never noticed that they never use uppercase letters. 😂 "Bundled npm packages" will work perfectly I think, avoiding the capital P as it seems out of place now. :P

Reply:
Okay, I’ve updated it.